base_tier_validation_correction/models/tier_correction.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class TierCorrection(models.Model):
+    _name = "tier.correction"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _description = "Tier Review Correction"
+    _order = "id desc"
+
+    name = fields.Char(
+        string="Description",
+        required=True,
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    model_id = fields.Many2one(
+        comodel_name="ir.model",
+        string="On Model",
+        domain=lambda self: [
+            (
+                "model",
+                "in",
+                self.env["tier.definition"]._get_tier_validation_model_names(),
+            )
+        ],
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    model = fields.Char(related="model_id.model", index=True, store=True)
+    correction_type = fields.Selection(
+        selection=[
+            ("reviewer", "Reassign Reviewer(s)"),
+        ],
+        string="Correction Type",
+        default="reviewer",
+        required=True,
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    name_search = fields.Char(
```

Comment:
Can you change `name_search`'s name to other name ?

I found the error when I click prepare button
![Selection_041](https://user-images.githubusercontent.com/24691983/129355386-09b10ffd-fdd4-436a-a017-09f6cd00b777.png)

My system installed module `base_name_search_improved` and `base_tier_validation_correction`